pkgs/applications/science/logic/formula/default.nix

```
@@ -0,0 +1,35 @@
+{ lib, stdenv, fetchFromGitHub, buildDotnetModule, dotnet-sdk_5 }:
+
+buildDotnetModule rec {
+  pname = "formula-dotnet";
+  version = "2.0";
+
+  src = fetchFromGitHub {
+    owner = "VUISIS";
+    repo = "formula-dotnet";
+    rev = "60dd3d91d931a795fa9fe6b6054f63e03cec9d28";
+    sha256 = "sha256-7odYV4FtImzX3vMUfO7sPlXkw6mOkZOFbnQLpJDXJc4=";
+  };
+
+  nugetDeps = ./nuget.nix;
+  projectFile = "Src/CommandLine/CommandLine.csproj";
+
+  makeWrapperArgs = "--set PATH ${dotnet-sdk_5}/bin";
+
+  installPhase = ''
+    runHook preInstall
+
+    makeWrapper ${dotnet-sdk_5}/bin/dotnet $out/bin/formula \
+      --add-flags "$out/bin/formula"
```

`$out/bin/formula` does not exist yet, that is what this wrapper will become. Because of that you cant use it as a flag, since itll just be referencing itself.
```suggestion
    makeWrapper ${dotnet-sdk_5}/bin/dotnet $out/bin/formula \
      --add-flags "$out/lib/formula-dotnet/CommandLine.dll"
```